examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,196 @@
+            // cameraPositions array stores the successive camera positions.
+            // It is used to compute the distance ran at each travel step.
+            let cameraPositions = [];
+            // Camera speed in meters per second
+            let cameraSpeed = 500 / 3.6;
+            // Boolean which states if the camera is performing a travel animation or not
+            let travelOn = false;
+
+            function saveCurrentCameraTransformOptions() {
+                // Retrieve camera current transform options
+                const cameraTransformOptions = itowns.CameraUtils.getTransformCameraLookingAtTarget(
+                    view,
+                    view.camera.camera3D,
+                );
+
+                // Save camera current transform options
+                cameraTransformOptions.coord.as(view.referenceCrs, cameraTransformOptions.coord);
+
+                if (cameraPositions.length > 0) {
+                    // Save the distance between the previous and the current picked points
+                    cameraTransformOptions.distance = view.camera.camera3D.position.distanceTo(
+                        cameraPositions[cameraPositions.length - 1],
+                    );
+                    cameraTransformOptions.easing = itowns.CameraUtils.Easing.Linear.None;
+                }
+
+                // Add the camera transform options and its position to the travel step arrays
+                cameraTravel.push(cameraTransformOptions);
+                cameraPositions.push(view.camera.camera3D.position.clone());
+
+                addCameraHelper()
+            }
+
+            function animateCamera() {
+                const camera = view.camera.camera3D;
+
+                // If travel is off, begin travel
+                if (!travelOn) {
+                    travelOn = true;
+                    // The travel time of each step is computed from the distance ran in the step and from the value of
+                    // cameraSpeed at the time of the travel.
+                    cameraTravel.forEach(function (item, index) {
+                        cameraTravel[index].time = (cameraTravel[index].distance || 0) / cameraSpeed * 1000;
+                    });
+
+                    // Return a promise for camera animation
+                    return itowns.CameraUtils.sequenceAnimationsToLookAtTarget(
+                        view,
+                        view.camera.camera3D,
+                        cameraTravel,
+                    ).then(() => {
+                        travelOn = false;
+                    });
+                }
+
+                // If travel is on, interrupt it and moves the camera to its initial placement :
+                itowns.CameraUtils.stop(view, view.camera.camera3D);
+                return itowns.CameraUtils.animateCameraToLookAtTarget(
+                    view,
+                    view.camera.camera3D,
+                    initialPlacement,
+                );
```

Comment:
could you remove the animate?

Reply:
I removed it. Now, if a user presses `t` while already having the camera traveling, the travel will just stop without moving the camera to its initial position.